packages/vx-tooltip/package.json

```diff
@@ -35,7 +35,7 @@
     "prop-types": "^15.5.10"
   },
   "peerDependencies": {
-    "react": "^15.0.0-0 || ^16.0.0-0"
+    "react": "^16.8.0-0"
```

Comment:
Why not bump it to “>= 16.8.0 <= 16.12.0”? Sure later versions are also supported, right?

Reply:
This will match any minor versions above and including `16.8`, you can try it [here](https://semver.npmjs.com/).

Reply:
Ah, I must’ve confused caret and tilde. Thank you!